src/plugins/conditionals/README.md

```diff
@@ -103,20 +103,19 @@ sudo kdb mount main.ini /examples/conditionals ni
 sudo kdb mount sub.ini /examples/conditionals/sub ini
 
 # mount conditionals as global plugin
-sudo kdb global-mount conditionals
+sudo kdb global-mount conditionals || $(exit 0)
```

Comment:
Ahh, perfect! #1301 would have need a rebase anyway. Why did you only pick 4 out of the 5 commits?